src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,49 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+  if((typeof text ) != "string"){
+        throw "Please enter a valid string";
+    }
+    text=text.split("");
+    let abc1 = ["A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z"];
+    let abc = ["a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z"];
```

Comment:
Consider

```js
const alphabets = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
const alphabetsUpperCaseArray = alphabets.split();
const alphabetsLowerCaseArray = alphabets.toLowerCase().split();
```

The variable names are clearer. Also, harder to miss one alphabet.